lib/c/strncmp.zig

```diff
@@ -0,0 +1,30 @@
+const builtin = @import("builtin");
+const std = @import("std");
+const common = @import("common.zig");
+
+comptime {
+    @export(&strncmp, .{ .name = "strncmp", .linkage = common.linkage, .visibility = common.visibility });
+}
+
+fn strncmp(s1: [*:0]const c_char, s2: [*:0]const c_char, n: usize) callconv(.c) c_int {
```

Comment:
for the impl I think you can use `std.mem.orderZ`. not that it matters right now but it will be nice in case it grows SIMD features, such as how indexOfScalar did.